openapi.yaml

```diff
@@ -1590,6 +1675,96 @@ components:
           "protocol_parameters": { "k": 5, "m": 100, "phi_f": 0.65 }
         }
 
+    StakeDistribution:
+      description: The list of participants in the Cardano chain with their associated stake
+      properties:
+        code:
+          type: string
+        text:
+          type: integer
+      example:
+        {
+          "pool15ka28a4a3qxgcgh60wavkylku4vqjg385jezsrqxlafyrhahf02": 1192520901428,
+          "pool1aymf474uv528zafxlpfg3yr55zp267wj5mpu4qt557z5k5frn9p": 1009503382720
+        }
+
+    CardanoStakeDistributionListMessage:
+      description: CardanoStakeDistributionListMessage represents a list of Cardano stake distribution
+      type: array
+      items:
+        type: object
+        additionalProperties: false
+        required:
+          - epoch
+          - hash
+          - certificate_hash
+          - created_at
+        properties:
+          epoch:
```

Comment:
Can you explain which epoch is used here for clarity?
Same for the other message below.